clusters/app.ci/prow-monitoring/mixins/_prometheus/dptp_alerts.libsonnet

```diff
@@ -160,6 +160,33 @@
           }
         ],
       },
+      {
+        name: 'openshift-priv-image-building-jobs-failing',
+        rules: [
+          {
+            alert: 'openshift-priv-image-building-jobs-failing',
+            expr: |||
+             (
+               sum(
+                 rate(prowjob_state_transitions{job="prow-controller-manager",job_name=~".*-images",org="openshift-priv",state="success"}[12h])
+               )
+               /
+               sum(
+                 rate(prowjob_state_transitions{job="prow-controller-manager",job_name=~".*-images",org="openshift-priv",state=~"success|failure|aborted"}[12h])
+               )
+             )
+             < 0.95
+            |||,
+            'for': '1m',
+            labels: {
+              severity: 'critical',
+            },
+            annotations: {
+              message: 'openshift-priv image-building jobs are failing at a high rate. Check on <https://deck-internal-ci.apps.ci.l2s4.p1.openshiftapps.com/?job=*-images|deck-internal>.',
```

Comment:
nit: 
```suggestion
              message: 'openshift-priv image-building jobs are succeeding at a low rate. Check on <https://deck-internal-ci.apps.ci.l2s4.p1.openshiftapps.com/?job=*-images|deck-internal>.',
```

Reply:
"succeeding at a low rate" is what we are literally checking now, but what we care about is that they are "failing at a high rate". We just had to group multiple states into "failure" so we check the inverse instead, but I still think this messaging makes more sense as is.

Reply:
Got it.